tests/baselines/reference/parserComputedPropertyName33.errors.txt

```diff
@@ -14,7 +14,7 @@ tests/cases/conformance/parser/ecmascript6/ComputedPropertyNames/parserComputedP
          ~~
 !!! error TS2304: Cannot find name 'e2'.
                ~
-!!! error TS1005: ';' expected.
+!!! error TS1068: Unexpected token. A constructor, method, accessor, or property was expected.

```

Comment:
This actually seems like the rare case where we `';' expected.` might be better. If I’m understanding this test correctly, the idea is that

```js
class C {
  [e] = 0
  [e2]() {}
}
```
is actually equivalent to
```js
class C {
  [e] = 0[e2]() {}
}
```

which, ignoring the semantic errors around `e` and `e2` themselves, is valid right up until the `{}`:

<img width="191" alt="image" src="https://user-images.githubusercontent.com/3277153/113045624-ec6fe000-9153-11eb-9c4f-1caa5fb122bc.png">

So after the `()`, which are actually the parens of a call expression in the initializer, I’m not sure we _are_ expecting “a constructor, method, accessor, or property.” We’re expecting either the end of the declaration (a semicolon), or something chaining onto the end of that expression.

Reply:
Super! I took a stab at adding a specific error message like _"A new function block or object is not allowed after this class member property. Is a semicolon missing after the property's value?"_ but didn't see a clean way to stop it from also firing in cases like:

```ts
class C {
    [e] = 0[e2](); {}
  }
```

An improvement set for another issue+PR, perhaps.